PersistentSwift/Classes/Attributes.swift

```diff
@@ -24,7 +24,7 @@ public protocol PSJSONAPIProperty: class {
     /// Setup relationships that are included inside of the JSON response
     ///
     /// - Parameter json: the "included" dictionary inside of the json Response
-    func addFromIncluded(_ json: JSON)
+    func addFromIncluded(_ json: JSON, objStore: PSServiceModelStore)
```

Comment:
"addFromIncluded" seems ambiguous, would "addRelationshipsFromIncluded" be a better name?